code/FIJI/batch_interactive_crop_savetif.v.0.0.ijm

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
You probably don't need to add the versions to these files. They're most likely unnecessary. If you want you can add it as a comment, but git is effectively versioning these.